blockmanager.go

```diff
@@ -330,6 +330,10 @@ type blockManager struct {
 	cachedCurrentTemplate *BlockTemplate
 	cachedParentTemplate  *BlockTemplate
 	AggressiveMining      bool
+
+	// The follownig field are used to filter duplicate block announcements.
```

Comment:
typo: follownig

Also use fields instead of field?

Reply:
Updated.